pc/core/src/main/scala/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -103,18 +104,34 @@ case class ScalaPresentationCompiler(
       global.typedTreeAt(pos).symbol.fullName
     }
   }
+  val whitelist = Array[Array[String]](
+    Array("scalamacros", "paradise"),
```

Comment:
anything that's been tested with pcplod should be fine. That includes `"org.scalaz", "deriving-plugin"`.

I wouldn't say that `paradise` is particularly PC friendly... it's one of the worst offenders.

Reply:
This is still a WIP and quite hacky, I shouldn't have opened a PR yet tbh. Will need more thorough coverage of plugins.

Reply:
That's what https://github.com/ensime/pcplod was designed for, so the plugins could be tested in their repository.